api/numaresourcesoperator/v1/numaresourcesoperator_types.go

```diff
@@ -189,3 +190,8 @@ func (ngc *NodeGroupConfig) ToString() string {
 	}
 	return ""
 }
+
+func (nro *NUMAResourcesOperator) IsCustomPolicyEnabled() bool {
+	_, ok := nro.Annotations[features.ForceCustomSELinuxPolicyAnnotation]
+	return ok
+}
```

Comment:
this should be a free function in a different package, perhaps `internal/api/annotations` again

Reply:
The rationale is that all the magic annotation constants should not be exposed in the public packages. Users will see only the magic strings, but they won't be part of the API contracts now (preferibily, never)

Reply:
Done. Thanks!